wagtail/snippets/tests/test_snippets.py

```diff
@@ -3997,16 +3997,16 @@ def test_with_live_and_draft_status(self):
         # Should show the "live version" status tag for the published revision
         self.assertContains(
             response,
-            '<span class="status-tag primary">Live version</span>',
-            count=1,
+            '<span class="w-status w-status--primary">Live version</span>',
+            count=0,
             html=True,
         )
 
         # Should show the "current draft" status tag for the draft revision
         self.assertContains(
             response,
-            '<span class="status-tag primary">Current draft</span>',
-            count=1,
+            '<span class="w-status w-status--primary">Current draft</span>',
+            count=0,
```

Comment:
```suggestion
            count=1,
```

Not sure why these counts were change, will revert this change.